code/modules/mob/living/carbon/monkey/life.dm

```diff
@@ -30,6 +30,9 @@
 
 /mob/living/carbon/monkey/handle_mutations_and_radiation()
 	if(radiation)
+		if(prob(max(0, radiation - RAD_MOB_MUTATE) / 25))

```

Comment:
I imagine we should add a variable one line above and then check if such variable is greater than 0 before running prob() every tick.